### DISCUSSION TITLE
How to use Adobe Experience Platform - Analytics Cordova plugin in Capacitor

### DISCUSSION THREAD

**Discussion by Arunchinnapparaj04** [ionic-team/capacitor#6143] [Q&A]
I am trying to implement Adobe Analytics in Capacitor but it has only cordova  plugins.

cordova plugin add https://github.com/adobe/cordova-acpanalytics.git

I Tried using the below commands to use in capacitor but no luck.

npm install https://github.com/adobe/cordova-acpanalytics.git
npm install @ionic-native/cordova-acpanalytics

**Comment by Thomas196-Sharon** [comment]
Hey, Let me know when should I reach out. Are the issues resolved yet?
Please reply at sharon@plaxonic.com
